documentation/restarting.markdown

```diff
@@ -35,10 +35,11 @@ There are three ways to trigger your Puppet Server environment to refresh and pi
 ### Changes applied after a HUP signal or full Server restart
 
 * Changes to Puppet Server [configuration files](./configuration.html) in its `conf.d` directory.
+* Changes to the CA CRL file.  For example, a `puppet cert clean`
 
 ### Changes that require a full Server restart
 
 * Changes to JVM arguments, such as [heap size settings](./tuning_guide.html#jvm-heap-size), that are typically configured in your `/etc/sysconfig/puppetserver` file.
 * Changes to [`bootstrap.cfg`](./configuration.html#service-bootstrapping) to enable or disable Puppet Server's certificate authority (CA) service.
 
-For these types of changes, you must restart the process by using the operating system's service framework, for example, by using the `systemctl` or `service` commands.
\ No newline at end of file
+For these types of changes, you must restart the process by using the operating system's service framework, for example, by using the `systemctl` or `service` commands.
```

Comment:
That's what I thought but I spent a few minutes trying to mess with that and it didn't seem to do anything.

Anyway, if you are ok with this as-is, you can merge it and I'll deal with merging up to the other branches.